server/src/main/java/org/opensearch/action/admin/indices/stats/CommonStatsFlags.java

```diff
@@ -100,6 +101,9 @@ public CommonStatsFlags(StreamInput in) throws IOException {
             includeCaches = in.readEnumSet(CacheType.class);
             levels = in.readStringArray();
         }
+        if (in.getVersion().onOrAfter(Version.CURRENT)) {
```

Comment:
why did you change version check to Version.CURRENT, it should 3.0.0

Reply:
Reverted